### DISCUSSION TITLE
Question: About Coyote supporting .NET Core

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/coyote#387] [Q&A] [answered]
Hello there,

I was working elsewhere in the depths while the Microsoft Open Source movement was going on. 

My current team has work that is being done in a large scale Azure environment. Does Coyote have .NET Core support or have plans to have .NET Core support? 

Does Coyote currently support testing new async await features introduced in 4.5?

**Comment by pdeligia** [comment, marked as answer]
Hi @mtmulch, Coyote has indeed .NET Core support. We support up to the latest .NET 6 release, you can find the list of supported targets here: https://www.nuget.org/packages/Microsoft.Coyote/#supportedframeworks-body-tab.

Coyote also indeed supports `async`/`await` from .NET Framework 4.5, but this C# lang feature and version was [released many years ago](https://devblogs.microsoft.com/dotnet/async-in-4-5-worth-the-await/), not sure what you mean by new?

**Comment by ghost** [comment]
@pdeligia Thank you for the reply. It was more so in reference to adoption of the feature. I will look closer into the tool at another point in time.
